test/plugin/test_cache_stats.rb

```diff
@@ -0,0 +1,39 @@
+#
+# Fluentd Kubernetes Metadata Filter Plugin - Enrich Fluentd events with
+# Kubernetes metadata
+#
+# Copyright 2015 Red Hat, Inc.
```

Comment:
Copyright 2017?

Reply:
Most likely can not update this as the other lint checks usually fail when I have tried to update the year